proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -138,6 +138,7 @@ message ValAddresses {
   option (gogoproto.goproto_stringer) = false;
   option (gogoproto.stringer)         = true;
 
+  // TODO should I add AddressString annotation for repeated string?
   repeated string addresses = 1;
```

Comment:
Just wonder if i may apply `cosmos.AddressString` annotation to `repeared string addresses`.